python/ruff-ecosystem/ruff_ecosystem/check.py

```diff
@@ -45,7 +45,7 @@
 )
 
 CHECK_DIAGNOSTIC_LINE_RE = re.compile(
-    r"^(?P<diff>[+-])? ?(?P<location>.*): (?P<code>[A-Z]{1,4}[0-9]{3,4}|SyntaxError:)(?P<fixable> \[\*\])? (?P<message>.*)"
+    r"^(?P<diff>[+-])? ?(?P<location>.*): (?P<code>[A-Z]{1,4}[0-9]{3,4}|invalid-syntax:)(?P<fixable> \[\*\])? (?P<message>.*)"
```

Comment:
We could probably change this to `[a-z\-]+` (I didn't test this pattern) to future prove (e.g. `io-error:`)